src/Packagist/WebBundle/Controller/ApiController.php

```diff
@@ -90,6 +90,9 @@ public function updatePackageAction(Request $request)
         if (isset($payload['project']['git_http_url'])) { // gitlab event payload
             $urlRegex = '{^(?:ssh://git@|https?://|git://|git@)?(?P<host>[a-z0-9.-]+)(?::[0-9]+/|[:/])(?P<path>[\w.-]+(?:/[\w.-]+?)+)(?:\.git|/)?$}i';
             $url = $payload['project']['git_http_url'];
+	    } elseif (isset($payload['repository']['html_url'])) { // gitea event payload
+            $urlRegex = '{^(?:ssh://git@|https?://|git://|git@)?(?P<host>[a-z0-9.-]+)(?::[0-9]+/|[:/])(?P<path>[\w.-]+(?:/[\w.-]+?)+)(?:\.git|/)?$}i';
```

Comment:
I'm unsure if this needs to be modified too.

@vtsykun 

the relevant JSON-Part looks like this:
```
{
  "repository": {
    "html_url": "https://xxxxx/test_repo",
    "ssh_url": "git@xxxxx/xxxxx/test_repo.git",
    "clone_url": "https://xxxxx/test_repo.git",
    "original_url": "",
  }
}
```